### PR TITLE
Add AF filter to amplicon report

### PR DESCRIPTION
**Pull request by Teja91**

### CODE REVIEW

File: resolwe_bio/tests/files/report_template.tex [hunk 1]
```diff
@@ -86,6 +86,8 @@
 \noindent
 {\lightfont {#PANEL#} | \today}
 
+\medskip
+\noindent Results have been filtered to exclude any putative variant detected below AF = {{#AF_THRESHOLD#}}\%. Variants below this threshold may be inspected manually in the unfiltered VCF files.
```
Comment (jkokosar) [inline resolwe_bio/tests/files/report_template.tex]: Is the wording correct? Variants _above_ this threshold ... ?

Comment (Teja91) [reply resolwe_bio/tests/files/report_template.tex]: It is. The report excludes variants below the threshold. If the user wants to see those that were excluded (so below threshold), they should check the VCF files.

**Comment by jkokosar** [comment]
Please add a CHANGELOG entry and rebase to the latest master

**Comment by JureZmrzlikar** [comment]
Rebased and added CHANGELOG.

**Pull request merged**